docs/hacking-guide/en/building.md

```diff
@@ -23,9 +23,9 @@ Install `NPM` using the instructions below
 
 The new npm-shrinkwrap.json should be written, commit it.
 
-#### Install npm On Fedora
+#### Install npm On Fedora 24
```

Comment:
I did a PR since proposing a change seemed easier than filling Jira and describing what I want to propose. For such simple change it is probably easier if you just do it the way you want than to make me amend it into the shape you want it in. That is, exactly how it ended up.